magpie/nn/models.py

```diff
@@ -47,7 +47,7 @@ def cnn(embedding_size, output_length):
     model.compile(
         loss='binary_crossentropy',
         optimizer='adam',
-        metrics=['accuracy'],
+        metrics=['top_k_categorical_accuracy'],
```

Comment:
Can you add some more info in the commit log to explain the change? Is there some online reference you can point to?